Add proposed structure

edmolyneux opened the pull request:
@amolpednekar @jdziek added a single object schema:

- should this be an array of mini transaction objects
- we're using unix timestamps for createdAt and updatedAt - should we use ISO8601 date-time strings?

amolpednekar commented:
1. It should be an array, yes
2. ISO-8601 makes sense, it is suggested by UK gov API standards as well - [Formatting dates and times in data
](https://www.gov.uk/government/publications/open-standards-for-government/date-times-and-time-stamps-standard)

edmolyneux commented:
OK, 

- structured as an array of items, 
- specified updatedAt and createdAt as ISO-86012 strings
- renamed to `pdtf-transaction-search-results.json'
- removed the `$schema` element
